src/encoded/locust/ff_locust.py

```diff
@@ -0,0 +1,57 @@
+import os
+import json
+import random
+from requests.auth import HTTPBasicAuth
+from locust import HttpLocust, TaskSet, task, between
+
+# IMPORTANT:
+# To best make use of this you'll want to pass in some authentication
+# To do this, create the file '<env>.json'. Navigate to the portal and
+# create an access key to be used via basic auth. Set 'username' and 'password'
+# in <env>.json to your ID and secret
+# config.json is provided. Use accordingly if you want to change routes, see 
+# locust.rst in docs
+
+
+def load_auth(loc):
```

Comment:
If this is going to be a general purpose load_auth, I'd rather it be in some separate utility package. If it's only used here, maybe it should be a static method on FFUserBehavior?

Reply:
I changed this to ``load_locust_auth`` to differentiate. I left a comment about this but FFUserBehavior is implicitly a static class so I'm not sure it makes sense to pull that function in